src/main/scala/org/apache/spark/sql/hive/llap/LlapContext.scala

```diff
@@ -154,6 +154,19 @@ class LlapCatalog(override val client: ClientInterface, hive: LlapContext)
     val tableWithQualifiers = Subquery(tableIdentifier.table, logicalRelation)
     alias.map(a => Subquery(a, tableWithQualifiers)).getOrElse(tableWithQualifiers)
   }
+
+  override def getTables(databaseName: Option[String]): Seq[(String, Boolean)] = {
+    val db = databaseName.getOrElse(client.currentDatabase)
+    val pattern = "*"
+    val result = new scala.collection.mutable.ArrayBuffer[(String)]
+    val conn = hive.connection
+    val rs = conn.getMetaData.getTables(null, db, pattern, null)
+    while(rs.next()) {
+      result+=rs.getString(3)
+    }
+    rs.close()
+    result.map(name => (name, false))
```

Comment:
for the hivemetastoreCatolog, it also set this as false. The reason is the false stands this table is tempview or not. Because the hive does not provide this, thus, the default is false.